libs/pages/elewa/social-impact/src/lib/components/social-impact/social-impact.component.scss

```diff
@@ -0,0 +1,51 @@
+.impact {
+    height: 100vh;
+    border-radius: 0 0 40px 40px;
+    // border: solid black;
+}
+.social {
+    display: flex;
+    align-items: center;
+    margin-bottom: solid black 20px;
+    padding: 7%;
+}
+img {
+    margin-right: 120px;
+}
+.title {
+    width: 50%;
+    
+}
+h2 {
+    font-family: var(--elewa-group-website-dmsans-bold);
+    font-size: 420%;
+}
+p {
+    font-family: var(--elewa-group-website-dmsans-regular);
+    font-size: 200%;
+}
+
+@media screen and (max-width: 820px) {
+    .impact {
+        margin: 0 0;
```

Comment:
The text should be left aligned on mobile and same width as image above.
spaces/indents should be 2 not 4